virtual-network.tf

```diff
@@ -28,7 +28,7 @@ resource "azurerm_subnet" "container_apps_infra_subnet" {
   resource_group_name  = local.resource_group.name
   address_prefixes     = [local.container_apps_infra_subnet_cidr]
 
-  service_endpoints = local.enable_container_app_blob_storage ? ["Microsoft.Storage"] : []
+  service_endpoints = local.container_apps_infra_subnet_service_endpoints
```

Comment:
Looks like we would lose the dynamic addition of `Microsoft.Storage` with this introduction? Can you adjust so that the storage endpoint is merged into `container_apps_infra_subnet_service_endpoints` if `enable_container_app_blob_storage` is `true` please?

Reply:
I believe I've got this covered in `locals.tf` line 21:

`  container_apps_infra_subnet_service_endpoints = concat(local.launch_in_vnet && local.enable_container_app_blob_storage ? ["Microsoft.Storage"] : [], var.container_apps_infra_subnet_service_endpoints)`

Unless you would like me to move this logic elsewhere?

In addition there's the potential problem of `Microsoft.Storage` being supplied in the var but also added here, but I'm unsure of best approach on fixing that.

Reply:
Sorted